pytest_jira.py

```diff
@@ -132,6 +141,14 @@ def pytest_addoption(parser):
     if os.path.exists('jira.cfg'):
         config.read('jira.cfg')
 
+    try:
+        verify = config.getboolean('DEFAULT', 'ssl_verification')
+    except ConfigParser.NoOptionError:
+        verify = True
+    except ValueError:
+        py.builtin.print_('Wrong argument for ssl_verification!')
```

Comment:
I'd think `log.warn()` (after appropriately initializing `log = logging.getLogger(__name__)` at the top of this module) might be more appropriate.  Why use `py.builtin.print_()`?